sdk/include/opentelemetry/sdk/metrics/meter.h

```diff
@@ -65,7 +65,7 @@ class Meter final : public opentelemetry::metrics::Meter
       nostd::string_view description = "",
       nostd::string_view unit        = "") noexcept override;
 
-  nostd::shared_ptr<opentelemetry::metrics::ObservableInstrument> CreateLongObservableGauge(
+  nostd::shared_ptr<opentelemetry::metrics::ObservableInstrument> CreateUInt64ObservableGauge(
```

Comment:
As earlier comment, this can be changed to `CreateInt64ObservableGauge`

Reply:
thanks, fixed